README.md

```diff
@@ -100,16 +100,36 @@ test a package with one command.
 
 # Instructions
 
-1. Clone this repository:
+1. Clone this repository and build `nixpkgs-update`:
     ```
     git clone https://github.com/ryantm/nixpkgs-update && cd nixpkgs-update
+    nix-build
     ```
-2. Get a list of outdated packages and place them in a `packages-to-update.txt` file in the root directory of this repository.
+
+2. Setup [hub](https://github.com/github/hub) and copy the oauth token to
+   `github_token.txt` in the root of this repository.
+
+3. To test your config, try to update a single package:
+   `./result/bin/nixpkgs-update --dry-run --additionalUpdates "X oldVer newVer"`,
+   where `X` is the attribute name, and `oldver` and `newVer` are the versions.
+
+If this works, you are now setup to hack on `nixpkgs-update`! To send a batch of
+updates, proceed as follows:
+
+4. Get a list of outdated packages and place them in a `packages-to-update.txt` file:
+
     ```
-    git clone https://github.com/ryantm/repology-api.git && cd repology-api
-    nix run nixpkgs.cabal2nix -c cabal2nix --shell --hpack . > shell.nix && nix-build shell.nix && result/bin/repology-api > ../packages-to-update.txt
+    ./result/bin/nixpkgs-update fetch-repology > packages-to-update.txt
     ```
-3. Return back `cd ..` and run the tool `nix run -c nixpkgs-update update`
+
+5. Run the tool in batch mode with `update-list`:
+
+    ```
+    ./result/bin/nixpkgs-update update-list --additionalUpdates ""
+    ```
+
+Note that the `--additionalUpdates` argument is currently mandatory, but does
+not need to have a real value.
 
 
 # Adding new dependencies / updating Cabal file
```

Comment:
Down in `Development tips` below, could you add a comment about how you do incremental builds, or maybe link to the section in `cabal2nix` that describes this?  The only thing I've been able to find out so far is just running a full `nix-build` on every change, which is OK-ish since this just has 19 translation units . . . but it'd be nice to actually build with something like cabal or stack :D

Reply:
I think this is what you are looking for: https://github.com/ryantm/nixpkgs-update/commit/1fe09df52b94c6104bd077055e1b5bd670185722